.gitignore

```diff
@@ -0,0 +1,2 @@
+TypeScript/bin/
```

Comment:
I don't think the TypeScript target uses a `bin` directory?

Reply:
TypeScript target uses `bin` directory for federated reactors, and shell scripts are generated there. Those shell scripts include commands for executing the RTI and federates in `.js`.